test/functional/services/find.ts

```diff
@@ -198,7 +198,7 @@ export async function FindProvider({ getService }: FtrProviderContext) {
       if (isDisplayed) {
         return descendant;
       } else {
-        throw new Error('Element is not displayed');
+        throw new Error(`Element "${selector}" is not displayed`);
```

Comment:
I was getting some flakiness in the tests, so I added this debug.

I think the flaky is coming from using `dashboardPanelActions.toggleContextMenu(savedSearchPanel)` to open the panel share menu.

Let's see if this keeps turning out to be flaky.